fix(learn): Improve note text on text shadow page (#40435)
mdn/content: files/en-us/learn_web_development/howto/solve_css_problems/add_a_text_shadow/index.md

sidebar: learn-how-to
---

In this guide, you'll learn how to add a shadow to any text on your page.

## Adding shadows to text

> In our [guide to adding a shadow to boxes](/en-US/docs/Learn_web_development/Howto/Solve_CSS_problems/Add_a_shadow), you'll learn how to add a shadow to any element on your page. However, that technique only adds shadows to the element's surrounding box. To add a drop shadow to the text itself, you need a different CSS property: {{cssxref("text-shadow")}}.

Reviewer: ```suggestion
Our [guide to adding a shadow to boxes](/en-US/docs/Learn_web_development/Howto/Solve_CSS_problems/Add_a_shadow) explains how to add a shadow to any element on your page. However, that technique only adds shadows to the element's surrounding box. To add a drop shadow to the text itself, you need a different CSS property: {{cssxref("text-shadow")}}.
```

Reviewer: Sounded a bit repetitive; you used the same phrasing earlier.

}
```

{{EmbedLiveSample("Text_shadow")}}

> [!NOTE]

> > When adding text shadows, you might unintentionally make the text difficult to read. Make sure that your choices keep your text readable and provide enough [color contrast](/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable/Color_contrast) for users who have difficulty reading low-contrast text.

Reviewer: ```suggestion
> When adding text shadows, you might unintentionally make the text difficult to read. Ensure that your choices offer sufficient [color contrast](/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable/Color_contrast) to keep your text readable.
```

Reviewer: The text seemed a bit long-winded; also:

- Keeping the text readable and having enough color contrast are not really two separate outcomes; the latter is a symptom of the former.
- Also, my suggestion gets rid of "for users who have difficulty reading low-contrast text", as it implies some people don't. All of us have trouble reading low-contrast text!
